.github/workflows/build.yml

```diff
@@ -124,9 +136,22 @@ jobs:
                 run: (cd tests/Application && yarn install)
 
             -
-                name: Prepare test application database
+                name: Create test application database
                 run: |
                     (cd tests/Application && bin/console doctrine:database:create -vvv)
+
+            -
+                name: Exclude duplicated migrations
+                if: matrix.sylius != '~1.8.0'
+                run: |
+                    (cd tests/Application && bin/console doctrine:migration:sync-metadata-storage)
+                    (cd tests/Application && bin/console doctrine:migrations:version 'Sylius\RefundPlugin\Migrations\Version20201130071338' --add)
```

Comment:
Could we somehow not execute them if we detect that those changes are already done? Currently, this would require to add those three migrations for every fresh plugin installation with Sylius 1.9.

Reply:
I don't know any way not to execute migrations if changes are already done. But I agree that it doesn't look as it should be and could be quite inconvenient during installation with Sylius 1.9. One of my thought was to move that duplicated migrations to `etc` directory and use them only for 1.8 (I've done sth like this in one of the commit: https://github.com/Sylius/RefundPlugin/pull/254/commits/6ba5b4502ad9c900a97de9d25da8ef28e720b014, but it didn't work as I wanted)

Reply:
I've added if statement to these migrations not to execute them if changes are already done